doc/classes/Texture2D.xml

```diff
@@ -12,6 +12,58 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="_draw" qualifiers="virtual const">
+			<return type="void" />
```

Comment:
Just a note in passing but as we recently moved the docs for constructors and operators to dedicated `<constructors>` and `<operators>` sections, it might make sense to do the same (in a follow-up) with a `<virtuals>` or something like this.

So we can make the docs a bit clearer with a separation between the API users can use, and the one users can extend/reimplement.